### ISSUE TITLE
Display application status in tray icon

### ISSUE THREAD

**Issue by yannxaver** [mkiol/dsnote#185]
Hello 👋🏻  First of all, thank you for building and sharing this application.

I automatically launch `dsnote` on startup like this:
```sh
# i3 config

exec_always --no-startup-id i3-msg 'workspace 9; exec alacritty -e 
flatpak run net.mkiol.SpeechNote --start-in-tray --verbose;'
```

I then start and stop dictation globally like this:
```sh
# i3 config

bindsym $mod+n exec 
flatpak run net.mkiol.SpeechNote --action start-listening-active-window

bindsym $mod+m exec 
flatpak run net.mkiol.SpeechNote --action stop-listening
```
This allows me to dictate into any currently selected text field. Would you say this is an appropriate configuration for using `dsnote` as a global on-demand dictation tool? Or would you suggest another configuration?

My only problem with this workflow: the tray icon does not display the current status of the application. So I have to remember to stop recording because there is no indicator. Have you considered adding an indicator to the tray icon which displays whether dictation is active or not?

**Comment by mkiol** [comment]
Hi. Thanks for sharing your setup. Yes, you are using the command line interface as intended :)

>  Have you considered adding an indicator to the tray icon which displays whether dictation is active or not?

It is already implemented in d722c9d4ecc9e6643015136e9ba7652c49d99a50 and will be included in the next release. You can test also it in [the beta release](https://github.com/mkiol/dsnote/issues/179).

**Comment by mkiol** [comment]
The new version 4.7.0 is out and available on flathub. 

New version brings:

- Status indication in the system tray icon. When using the system tray icon, statuses such as processing, listening, etc. are presented with an animated tray icon.

Video demo: https://youtu.be/cEht4Fts6Bo?feature=shared&t=265
